.github/workflows/ui_test.yml

```diff
@@ -16,28 +16,29 @@ jobs:
     timeout-minutes: 10
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - uses: actions/setup-node@v3
-      with:
-        node-version: 18
-    - name: Install dependencies
-      run: yarn
-    - name: Install Playwright Browsers
-      run: yarn playwright:install
-    - name: Run Playwright tests
-      run: yarn playwright:test
-      env:
-        VITE_PROJECT_ID: ${{ secrets.VITE_DEV_PROJECT_ID }}
-        VITE_EXPLORER_API_URL: ${{ secrets.VITE_EXPLORER_API_URL }}
-    - uses: actions/upload-artifact@v3
-      if: always()
-      with:
-        name: playwright-report
-        path: playwright-report/
-        retention-days: 7
-    - uses: actions/upload-artifact@v3
-      if: always()
-      with:
-        name: test-results
-        path: test-results/
-        retention-days: 7
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Install dependencies
+        run: yarn
+      - name: Install Playwright Browsers
+        run: yarn playwright:install
+      - name: Run Playwright tests
+        run: yarn playwright:test
+        env:
+          VITE_PROJECT_ID: ${{ secrets.VITE_DEV_PROJECT_ID }}
+          VITE_EXPLORER_API_URL: ${{ secrets.VITE_EXPLORER_API_URL }}
+          VITE_CI: true
```

Comment:
Added this env variable to make some conditional renderings for the CI. Doesn't seem perfect solution but need to invest more time in the notifications. As I see, Playwright doesn't have enough support on notifications yet. 

- See https://github.com/microsoft/playwright/issues/3301